nuxeo-business-days-management-core/src/main/java/org/nuxeo/business/days/management/checker/CSVHolidaysChecker.java

```diff
@@ -93,6 +80,15 @@ private void initDates(String fileName, boolean isEmbedded) {
 
     }
 
+    protected Reader getReader(String fileName, boolean isEmbedded) throws IOException {
```

Comment:
`@since` could be useful in case of backport tentative

Reply:
Not useful for non-public methods.